### ISSUE TITLE
 Unable to scroll up and down using the Dragon speech recognition software

### ISSUE THREAD

**Issue by ProdLead** [wekan/wekan#2499]
## Issue

**Here is the scenario in WeKan board

When there are multiple cards in a list, the user would like to move up and down to find a card of his interest. He cannot use mouse/keyboard due to motor disability. 
He has Dragon naturally speaking software installed in his system using which he gives command. But he is unable to scroll up or down. The card window has similar issue.**

**Comment by xet7** [comment]
@ProdLead 

Is there Dragon naturally speaking software commands for mouse scroll? Wekan uses mouse scroll to scroll up/down.

Is there difference in using:
- desktop browser full width mode
- desktop browser smaller width mobile mode, there currently moving with mobile phone finger is possible, maybe it could also work for Dragon naturally speaking?

Related #2081 and #1878

**Comment by ProdLead** [comment]
Here is the reply from the end user.

Yes, there are Dragon Naturally Speaking speech commands for scrolling up and down: "scroll up" and "scroll down", which I tried. It works fine with Windows applications as you generally have only one scrollbar for the window you are working on. However it doesn't work with the WeKan layout, with its many scroll bars on the one screen.

---------------------------
Could you explain your advice in detail?
1. Desktop browser full width mode -  I'm not sure what this is??????
2. Desktop browser smaller width mobile mode, there currently moving with mobile phone finger is possible, maybe it could also work for Dragon naturally speaking??????

****

**Comment by xet7** [comment]
@ProdLead 

I think those 1. and 2. are more related to Wekan bug #2492 so I think this is not about desktop/mobile mode. I try to ask using better way below.

1. I don't currently have Dragon Naturally Speaking software. Someone should buy license of Dragon Naturally Speaking software to me, so I could add support for it. If that software requires spefic headset, someone should also buy that for me. I do have some USB headset. If someone is interested in this, please email me x@xet7.org . There is also [some alternative Open Source speech recognition software](https://hackaday.com/2018/01/17/speech-recognition-for-linux-gets-a-little-closer/), I don't know would some of those work also.
2. Someone should pay for those work hours I use to add Dragon Naturally Speaking software support to Wekan. My hourly price is listed at https://wekan.team .
3. Your end user should give me list of Dragon Naturally speaking commands that should be added to Wekan. For example, is it only "scroll up" and "scroll down" ? What commands does end user use in other apps, so that same command should be added to work in Wekan too?

**Comment by xet7** [comment]
@ProdLead 

Does your Wekan end user still use Dragon Naturally Speaking? Or has end user changed to some other software? What version of software does end user currently use?

I'm currently researching how to fix this.
